tests/pyincore/analyses/roaddamage/test_roaddamage.py

```diff
@@ -63,7 +63,7 @@ def run_with_base_class():
     # test galveston hurricane road failure
 
     # road inventory for Galveston island
-    road_dataset_id = "60ba5ee94219ee25a56b6999"
+    road_dataset_id = "664e5812efdc9f1ed5dc2f7f"
```

Comment:
Could you also update the tests/pyincore/analyses/roaddamage/test_roaddamage_w_hazard_obj.py. It is using the same id which doesn't exist anymore.

Reply:
updated